ios/NewExpensify/AppDelegate.mm

```diff
@@ -0,0 +1,172 @@
+#import "AppDelegate.h"
```

Comment:
Probably want to review this with extra care – it was renamed from `AppDelegate.m` to `AppDelegate.mm` as RN migrated it from Objective-C to Objective-C++. However, we had a fair amount of custom code in `AppDelegate.m` that I ported over to this file manually.